config/environments/sit.rb

```diff
@@ -147,4 +147,7 @@
   # Metadata preview urls
   config.metadata_preview_js_url = ENV['metadata_preview_js_url']
   config.metadata_preview_css_url = ENV['metadata_preview_css_url']
+
+  #Feature toggle for using legacy order service
+  config.use_legacy_order_service = false
```

Comment:
should this be:
config.use_legacy_order_service = ENV['use_legacy_order_service'] || false
based on PR discussion.

Reply:
Done, set in application.rb.